frameworks/CSharp/aspnet/benchmark_config.json

```
@@ -203,7 +203,7 @@
       "notes": "",
       "versus": ""
     },
-    "mono": {
+    "default": {
```

Without those changes running test in vagrant fails early